sources/private/yarn-plugin-bud/sources/command/release.command.ts

```diff
@@ -136,26 +136,17 @@ export class Release extends Command {
    */
   public async wipeProxyDb() {
     this.log('Wiping previously published packages')
-
-    /**
-     * Wipe db records if they exist
-     */
-    const dbExists = await pathExists(`./storage/verdaccio/.verdaccio-db.json`)
-    if (!dbExists) return
-
-    const verdaccioDb = await readJson(
-      `./storage/verdaccio/.verdaccio-db.json`,
-    )
-    verdaccioDb.list = []
-    await writeJson(`./storage/verdaccio/.verdaccio-db.json`, verdaccioDb)
-
-    /**
-     * Wipe packages if they exist
-     */
-    const packagesExists = await pathExists(`./storage/verdaccio/packages`)
-    if (!packagesExists) return
-
-    await remove(`./storage/verdaccio/packages/@roots`)
+    try {
```

Comment:
i think it's easier and probably faster to just wrap the fs checks in a try/catch.

this is kind of janked right now and i don't know why. only effects me, though.